### DISCUSSION TITLE
No navigation when viewing linked content

### DISCUSSION THREAD

**Discussion by anthonywhite** [community/community#15034] [Mobile]
When the mobile app is opened from a link elsewhere e.g. Twitter, there is no way to navigate within GitHub, e.g. up or to home. All that is offered is the back arrow to return to the link.

**Comment by brianlovin** [comment]
@anthonywhite thanks for the note — I'm guessing you're using the Android app, is that correct?

**Reply by anthonywhite** [reply]
Correct. I guess further feedback might be to log the o/s when capturing feedback 🙂

**Reply by brianlovin** [reply]
Agreed, we should do that to save some time here. I'll open a separate tracking issue for that now 😅

**Comment by theetrain** [comment]
Thanks for acknowledging this and related breadcrumb/navigation threads currently in discussion.

I recently experienced a related problem: 

1. Open a few email links to a GitHub issue
2. Time passes, I eventually open the GitHub app to use it
3. I have to tap the "Back" (breadcrumb) button many times before being able to go to "Home" or "Notifications"

I'm sure the right solution is being discussed, and I encourage something that includes a persistent bottom navigation panel that way I can jump to Home or Notifications from anywhere.

**Comment by alialhammad1992** [comment]
Unfortunately, Github is slow in debugging the mobile app
